app/package.json

```diff
@@ -37,10 +42,11 @@
     },
     "extends": [
       "plugin:vue/recommended",
-      "eslint:recommended"
+      "eslint:recommended",
+      "@vue/typescript"
     ],
     "parserOptions": {
-      "parser": "@babel/eslint-parser"
+      "parser": "@typescript-eslint/parser"
     },
     "rules": {
       "vue/multi-word-component-names": "off"
```

Comment:
I would suggest adding these five rules for better formatting.

`"space-before-function-paren": 2,
 "object-curly-spacing": ["error", "always"],
 "array-bracket-spacing": ["error", "always"],
 "computed-property-spacing": ["error", "always"],
"quotes": ["error", "single"]`

Reply:
Thanks, added those rules in recent commit!